interfaces/builtin/network_manager.go

```diff
@@ -294,6 +294,24 @@ dbus (receive, send)
     peer=(label=###SLOT_SECURITY_TAGS###),
 `
 
+const networkManagerSendIntrospectionSnippet = `
```

Comment:
This should have ConnectedPlug in the name somewhere.

Reply:
Done